README.md

```diff
@@ -24,13 +24,29 @@ Psm assumes that application configuration is stored in a specific heirarchy in
 
 You can override the default path, by passing a hardcoded string in the `x-path-override` header.
 
+### Block value parser
+
+You can override the default parser by passing a `x-kv-block-parser: true` in the header. This will use parent nodes of the json as keys (see example below).
+
+#### eg. input json:
+
+`{ "keyOne": { "bgContactUsPhone": "+1 (214) 600 0090", "PhoneYYY": "+1 (214) 999999" }, "keyTwo": { "PRE_ORDER_SORT_OPTION": "false" }, "keyThree": { "API_ENDPOINT_CRP": { "uri": "/dt/api/crp-svc/US", "toggles": ["PRE_ORDER_SORT_OPTION", "BG_PHASE_2_RELEASE_2"] } } }`
```

Comment:
Can you sanitize this data a bit more?
Can you move this into a code block for formatting?: \```JSON